paddle/fluid/operators/conv_op.cc

```diff
@@ -209,6 +209,12 @@ void Conv2DOpMaker::Make() {
       .SetDefault(false);
   AddAttr<bool>("fuse_relu", "(bool, default false) Only used in mkldnn kernel")
       .SetDefault(false);
+  AddAttr<bool>("fuse_leaky_relu",
```

Comment:
Since Relu is leaky relu with alpha=0 , it would be good not to introduce  attribute "fuse_leaky_relu" and use "fuse_relu" along with "fuse_leaky_relu_alpha" to perform leaky relu and relu.

Reply:
That's a good point. Since I'm currently working on rewriting whole conv+activation pass creation strategy, I won't change it in this PR. Its main point is checking if it speeds up face model.

Reply:
Please remove the attr in this PR. Thanks very much!
The reason is that if you add the attr here, then all the models trained with the commit will have above attrs in there program_desc.

Reply:
If I will finish my general conv+activation pass by tommorow I will just close this PR and go with updated one since there is no point in merging old one then. However, I really would like to know if it works on face model before I put major changes.

Reply:
```
I0731 15:21:47.283124 25431 helper.h:326] ====== batch latency: 49.1396ms, number of samples: 1, sample latency: 49.1396ms, fps: 20.3502, data type: float ======
W0731 15:21:47.284312 25431 profiler.cc:89] CUDA CUPTI is not enabled

------------------------->     Profiling Report     <-------------------------

Place: CPU
Time unit: ms
Sorted by total time in descending order in the same thread

Event                        Calls       Total       Min.        Max.        Ave.        Ratio.
thread0::conv2d_transpose    400         3886.4      2.79948     45.771      9.71599     0.79294
thread0::conv2d              400         886.211     0.748841    4.3301      2.21553     0.180813
thread0::batch_norm          300         51.3999     0.120103    1.15955     0.171333    0.0104871
thread0::relu                300         40.7469     0.071005    0.297146    0.135823    0.00831358
thread0::load_combine        1           12.8519     12.8519     12.8519     12.8519     0.00262217
thread0::tanh                100         12.3476     0.11473     0.309273    0.123476    0.00251927
thread0::fetch               100         10.7564     0.098268    0.300906    0.107564    0.00219462
thread0::feed                100         0.54181     0.004738    0.006375    0.0054181   0.000110545
```
It seems no speedup on Model level.